### PR TITLE
feat: ipfs comparisons

### LINKED ISSUE #779: Page of "How is IPFS different from..."
* bittorrent
* storj
* arweave
* sia
* filecoin
* dat
* ???

### PR DESCRIPTION
**Pull request by DannyS03**
## Context
- Document on comparing IPFS to other [similar] storage solutions
- Incorporates some suggestions in the linked issue and utilizes a table to compare common parameters and features
- Unsure where the best place is for this to live; under case studies atm
- resolves #779

## Latest preview

Please view the latest Fleek preview [here](https://bafybeiece5dct2obrlaxebta2k7mi7fcqd2777e6gazxisb3smzzyjbyfy.on.fleek.co/concepts/comparisons/).

### CODE REVIEW

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

File: docs/concepts/comparisons.md
Comment (ElPaisano) [inline docs/concepts/comparisons.md:10]: Suggestion: This sentence 
```
This sets it apart from other solutions with a more specific focus or using different data storage mechanisms. For example:
```
reads a little strange. I think rephrasing it to something like
```
This sets it apart from other storage solutions that use different data storage mechanisms or have a more specific focus .  Similar solutions include:
```
would read better.

File: docs/concepts/comparisons.md
Comment (ElPaisano) [inline docs/concepts/comparisons.md:24]: Two comments:
- Phrasing suggestion "Note that all of these solutions use content-based addressing."
- Might be better to use a callout box here:
   ```
   :::callout
   Note that all of these solutions use content-based addressing. 
   ::::
   ```

File: docs/concepts/comparisons.md
Comment (ElPaisano) [inline docs/concepts/comparisons.md:16]: Would suggest a `:::callout` here

**Review by ElPaisano — CHANGES_REQUESTED** [review]
Hi @DannyS03 I like the idea of adding this info into the docs. 

Made a few review notes

RE location in the docs: I'm not sure how I feel about this page being in Case Studies. The way I understand a Case Studies section, it's a section of the docs where examples of applications / other things that use / integrate / build on the software (in this case, IPFS), so this page feels out of place in this section, since it's a comparison of IPFS to other technologies. I'd actually suggest adding it as a subsection on the /concepts/what-is-ipfs/ page instead or at least somewhere in the IPFS 101 subsection under Concepts. Open to thoughts here

File: docs/.vuepress/config.js
Comment (2color) [inline docs/.vuepress/config.js:272]: This isn't a case study. I would consider putting this in the IPFS 101 category.

File: docs/concepts/comparisons.md
Comment (2color) [inline docs/concepts/comparisons.md:17]: It's more commonly referred to as the Hypercore protocol nowadays. https://hypercore-protocol.org/

**Review by 2color — CHANGES_REQUESTED** [review]
I would recommend making a clear distinction between two main categories here:
- Crypto-economic networks like Filecoin and Arweave
- Protocols made for moving (routing & transferring) bytes around like BitTorrent, Hypercore, IPFS

I'd also split those into two different tables to avoid creating a misleading comparison between the two categories

Some illuminating resources: 
https://blog.mauve.moe/posts/protocol-comparisons
https://medium.com/decentralized-web/comparing-ipfs-and-dat-8f3891d3a603

File: docs/.vuepress/config.js
Comment (ElPaisano) [reply docs/.vuepress/config.js:272]: > I'd actually suggest adding it as a subsection on the /concepts/what-is-ipfs/ page instead or at least somewhere in the IPFS 101 subsection under Concepts. 

Seems like there's a shared consensus here @DannyS03. Can you move this out to IPFS 101?

File: docs/concepts/comparisons.md
Comment (ElPaisano) [reply docs/concepts/comparisons.md:6]: @2color created this issue https://github.com/ipfs/ipfs-docs/issues/1368 based on your callout. I also added your notion to the docs reorg notion

Comment (2color) [reply docs/concepts/comparisons.md:6]: Thanks, @ElPaisano. That's great

**Comment by 2color** [comment]
Btw, I should add that this page is very valuable. Might be worth also adding Git, which is also a distributed content-addressed system that has many parallels to IPFS.

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Comment by filecorgi** [comment]
- Image optimization came back clean!
- Vuepress build was successful!

**Review by ElPaisano — APPROVED** [review]

**Comment by ElPaisano** [comment]
@DannyS03 changes look good 👍 @2color looks like DannyS03 addressed your changes. Is this good to go from your end? If so, we can merge once you approve.

**Review by 2color — APPROVED** [review]

**Comment by DannyS03** [comment]
Thanks for the suggestions @ElPaisano @2color! To Daniel's point, let's continue keeping an eye out for this page and consider other solutions + comparisons.

**Pull request merged**
